tests/sentry/data_export/test_tasks.py

```diff
@@ -286,3 +343,23 @@ def test_discover_snuba_error(self, emailer, mock_query):
             assemble_download(de.id)
         error = emailer.call_args[1]["message"]
         assert error == "Internal error. Your query failed to run."
+
+    @patch("sentry.data_export.models.ExportedData.finalize_upload")
+    @patch("sentry.data_export.models.ExportedData.email_failure")
+    def test_discover_integrity_error(self, emailer, finalize_upload):
+        de = ExportedData.objects.create(
+            user=self.user,
+            organization=self.org,
+            query_type=ExportQueryType.DISCOVER,
+            query_info={"project": [self.project.id], "field": ["title"], "query": ""},
+        )
+        finalize_upload.side_effect = IntegrityError("test")
+        with self.tasks():
+            assemble_download(de.id)
+        error = emailer.call_args[1]["message"]
+        assert error == "Failed to save the assembled file."
+
+
+class MergeExportBlobsTest(TestCase, SnubaTestCase):
+    def test_task_persistent_name(self):
+        assert merge_export_blobs.name == "sentry.data_export.tasks.merge_blobs"
```

Comment:
What's the purpose of this test? Is it important that name matches up?

Reply:
Not entirely sure, this was asserted for the original task. I assumed it was just to make sure no one accidentally changes it?